crates/swc/tests/tsc-references/privateNameComputedPropertyName2(target=es2022).2.minified.js

```diff
@@ -1,7 +1,5 @@
 //// [privateNameComputedPropertyName2.ts]
-let getX;
-class A {
+console.log((new class {
     #x = 100;
-    [(getX = (a)=>a.#x, "_")]() {}
-}
-console.log(getX(new A));
+    _() {}
+}).#x);
```

Comment:
Oh, this is wrong

Reply:
Currently swc cannot correctly handle class scope in analyzer, I'd prefer to fix it later.
Example
```js
export class A {
   foo() {
      var o = this
      return class {
        x = console.log(o)  
      }
   }
}
```